### PR TITLE
TC-630 Add react testing library

### PR DESCRIPTION
**Pull request by kscoulter**
Adds react testing library and first passing test in the new testing paradigm.

### CODE REVIEW

**Review by peggles2 — CHANGES_REQUESTED** [review]
Can you add a description for this pull request? 
Just a few comments.

File: solution/ui/src/pages/index.test.js [hunk 1]
```diff
@@ -1,27 +1,33 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+// import { shallow } from 'enzyme'
```
Comment (peggles2) [inline solution/ui/src/pages/index.test.js:2]: We can remove this comment as well

File: solution/ui/src/pages/index.test.js [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import React from 'react'
 import { shallow } from 'enzyme'
```
Comment (peggles2) [inline solution/ui/src/pages/index.test.js:2]: Are we using enzyme anymore here?

**Review by peggles2 — CHANGES_REQUESTED** [review]
Just one more tweak :)

File: solution/ui/src/pages/index.test.js (hunk 2 above)
Comment (kscoulter) [reply solution/ui/src/pages/index.test.js:2]: yes, acceptance criteria for this ticket is one passing test. I didn't rewrite them all for this part.

**Review by peggles2 — APPROVED** [review]
Nice work!

**Pull request merged**